CHANGELOG.md

```diff
@@ -2,2020 +2,2027 @@
 
```

Comment:
Please revert all unnecessary changes to the `changelog`.

Reply:
Fixed.